app/helpers/custom_links_helper.rb

```diff
@@ -0,0 +1,18 @@
+module CustomLinksHelper
+  def custom_links
+    require 'json'
+    result = ''
+    source = CustomSetting['custom_content.header_menus']
+    begin
+      json = JSON.parse(source)
+      json.each do |j|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'j' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->